pkg/tsdb/cloudwatch/metric_data_query_builder.go

```diff
@@ -88,12 +88,11 @@ func buildSearchExpression(query *cloudWatchQuery, stat string) string {
 	}
 
 	if query.MatchExact {
-		schema := query.Namespace
+		schema := fmt.Sprintf(`"%s"`, query.Namespace)
```

Comment:
```suggestion
		schema := fmt.Sprintf("%q", query.Namespace)
```

I would just using `%q` instead that prints a double-quoted string safely escaped with Go syntax.